src/protocol/securemul.rs

```diff
@@ -88,134 +88,6 @@ impl<'a, N: Network> SecureMul<'a, N> {
 #[derive(Error, Debug)]
 pub enum Error {}
 
-/// Module to support streaming interface for secure multiplication
```

Comment:
Why did you remove this whole streaming multiplication thing?

Reply:
ah yea, sorry I mentioned this in the commit, but did not mention in the PR description. It is basically useless behaviour for us because that is not how we use multiplications. Cost of maintaining it raises - for example there was a hidden bug here: `stream_element_idx` was moved inside the closure so it was never incremented more than once. I tried to fix it but there is no easy way to do that (maybe `Arc<AtomicU32>>` could be a fix). 

Given that we don't observe this pattern, this is dead code and I think it should be removed, but we could reconsider that if there is an argument for keeping it around